src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx

```diff
@@ -195,7 +196,15 @@ const CheckInConfirmation = props => {
     );
   };
 
-  return isLoading ? renderLoadingMessage() : renderConfirmationMessage();
+  if (
```

Comment:
I noticed the `isLoading` is gone from this check. Is it unnecessary now?

Reply:
@wullaski yeah, it can't be set true by default so I used the new approach. I left it in the custom hook because I think we may need it later if we call the API after showing the confirmation screen.